makefishmeet.sh

```diff
@@ -0,0 +1 @@
+make && cp fishmeet/*.* ./libs
```

Comment:
can this be part of the `Makefile` so that we don't need to run this as additional step? (thus no need to update the instructions in `fishmeet-server` repo)